pkg/chunk/cache/fifo_cache_test.go

```diff
@@ -72,18 +75,55 @@ func TestFifoCache(t *testing.T) {
 	}
 }
 
-func TestFifoCacheExpiry(t *testing.T) {
-	c := NewFifoCache("test", FifoCacheConfig{Size: size, Validity: 5 * time.Millisecond})
+func TestFifoCacheEvictionExpiry(t *testing.T) {
+	c := NewFifoCache("test2", FifoCacheConfig{Size: 3, Validity: 5 * time.Millisecond})
 	ctx := context.Background()
+	memorySz := int(unsafe.Sizeof(cacheEntry{})) * 3
 
-	c.Put(ctx, []string{"0"}, []interface{}{0})
+	assert.Equal(t, testutil.ToFloat64(c.entriesAdded), float64(0))
```

Comment:
Can we use `require.Equal` here not to add one more dependency? Or use this asset package: https://godoc.org/github.com/stretchr/testify/assert

Reply:
`assert` is more appropriate, but as Goutham outlined the wrong package was imported. The correct one is:
`github.com/stretchr/testify/assert`

Please cleanup `vendor/` and `go.mod` once done.